doc/level0.rst

```diff
@@ -119,62 +115,70 @@ the following sections for header format definitions and parsing.
 Deriving a Parsed Autocrypt Header from a Message
 -------------------------------------------------
 
-The ``Autocrypt:`` header MUST have the following format::
+The ``Autocrypt`` header has the following format::
 
     Autocrypt: to=a@b.example.org; [type=(p|_*);] [prefer-encrypted=(yes|no);] key=BASE64
 
-Where the value of ``key`` includes a Base64 representation of a
-minimal key. For now we only support ``p`` as the type, which
-represents a specific subset of OpenPGP (see the the next section).
-``key`` MUST be the last attribute.
+The ``to`` attribute indicates the single recipient address this header is
+valid for. In case this address differs from the one the MUA considers the
+sender of the e-mail in parsing, which will usually be the one specified in the
+``From`` header, the entire header MUST be treated as invalid.
+
+The ``type`` and ``key`` attributes specify the type and data of the key
+material.  For now the only supported type is ``p``, which represents a
+specific subset of OpenPGP (see the next section), and is also the default.
+Headers with an unknown ``type`` MUST be treated as invalid.  The value of the
+``key`` attribute is a Base64 representation of the public key material.  For
+ease of parsing, the ``key`` attribute MUST be the last attribute in the
+header.
 
-``prefer-encrypted`` indicates that agents should default to
+The ``prefer-encrypted`` attribute indicates whether agents should default to
 encrypting when composing e-mails to this recipient.
 If ``prefer-encrypted`` is not set,
 the value of ``prefer-encrypted`` is ``nopreference``.
 If ``prefer-encrypted`` is set, but neither ``yes`` nor ``no``,
 the MUA must skip the header as invalid.
 
-Additional attributes unspecified here are also possible before the
-``key`` attribute.  If a MUA encounters an unknown attribute, if the
-attribute name starts with an underscore (``_``), this is a
-"non-critical" attribute.  The MUA MUST ignore the non-critical
-attribute and continue parsing the header.  If the unknown attribute
-name does *not* start with an underscore, it is a "critical"
-attribute, and the MUA must skip the header as invalid.
+Additional attributes unspecified here are also possible before the ``key``
+attribute.  If an attribute name starts with an underscore (``_``), it is a
+"non-critical" attribute.  The MUA SHOULD ignore non-critical attributes, but
+MUST treat the entire header as invalid if it encounters a "critical" attribute
+it doesn't support.
```

Comment:
This paragraph seems weaker than the original: the original is about encountering unknown attributes, but the replacement seems to suggest that MUAs SHOULD ignore non-critical attributes it is aware of.